src/main/java/frc/robot/constants/FeatureFlags.java

```diff
@@ -2,7 +2,7 @@
 
 public final class FeatureFlags {
     public static final boolean runDrive = true;
-    public static final boolean runVision = true;
+    public static final boolean runVision = false;
```

Comment:
We might as well leave this true

Reply:
yeah, i had turned it off as it messes with positioning in sim, so it was throwing off the alignment. but ill put it back on